build/constants.yaml

```diff
@@ -47,6 +47,7 @@
     dot-net-60-sdk-version: 6.0.412
     dot-net-60-rtm-sdk-version: 6-0-100-rtm-21527-11
     dot-net-70-sdk-version: 7.0.306
+    dot-net-80-sdk-version: 8.0.100-preview.7.23376.3
     runtime-versions:
       3.0: debian-buster
```

Comment:
Let's make sure to add a section for `8.0: debian-bookworm` in this `runtime-versions` dictionary if we're producing a runtime image for 8.0

Reply:
Updated thanks!